src/components/AppFooter/Livechat.tsx

```diff
@@ -1,5 +1,6 @@
 import { useGrowthbookGetFeatureValue, useLiveChat } from '@/hooks/custom-hooks';
 import useFreshChat from '@/hooks/custom-hooks/useFreshchat';
+import useIntercom from '@/hooks/custom-hooks/useIntercom';
```

Comment:
nit: can we also export these in the index file of custom-hooks? then you can do this
```suggestion
import { useIntercom, useFreshChat, useGrowthbookGetFeatureValue, useLiveChat } from '@/hooks/custom-hooks';
```